services/ns_net/BUILD.gn

```diff
@@ -4,6 +4,10 @@
 
 source_set("ns_net") {
   sources = [
+    "allocation_builder.cc",
+    "allocation_builder.h",
+    "data_pipe_flusher.cc",
+    "data_pipe_flusher.h",
```

Comment:
Yeah, I didn't quite know where to put this stuff so just put it there for the time being. Will move it to `glue`.